lib/LocalVerifiableCredentialStore.js

```diff
@@ -20,7 +20,8 @@ export class LocalVerifiableCredentialStore {
           'content.id',
           'meta.parentId',
           'meta.profileId',
-          'meta.displayable'
+          'meta.displayable',
+          'content.type'
```

Comment:
This index doesn't seem like it would be used by the queries in the code below. I'm presuming that the API here will create a compound index with all of the given fields. But maybe that's not how it works?

If that is how it works, then all of the fields before `content.type` would have to be provided when `content.type` is provided in order for the index to be used. But the `find` code below seems to accept any number of these different fields independently -- and it doesn't even take `content.id`. That would imply the index would never be used at all via `find`. I would expect that multiple indexes would be needed here -- one to cover `content.id` and others to cover the other queries we want to support.

Does this do what it's supposed to do?

Reply:
Yes this does what it is supposed to do. I am a little confused by your comment and not sure what I need to do in order to accommodate your request.

Reply:
Actually I think you are right. If there is an array of types, it only gets the first 1. I thought this was working because there is a fallback to remote afterwards and it was getting the type I was using as an example from remote instead of local.

Here is the pouchDB docs for query: https://pouchdb.com/api.html#query_index

I am unsure how to get it to properly handle and array of types.

Reply:
Alright I updated the code and it seems to be able to handle and array of types now with the latest fix I pushed.